src/iap/shared/interceptors/http-token.interceptor.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
```

Comment:
:point_right: :point_left: 
```suggestion
import { Injectable } from '@angular/core';
import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';

import { Observable } from 'rxjs';
```